src/ShallowTraversal.js

```diff
@@ -27,6 +27,9 @@ export function childrenOfNode(node) {
 
 export function hasClassName(node, className) {
   let classes = propsOfNode(node).className || '';
+  if (typeof classes !== 'string') {
+    classes = classes.toString();
+	}
```

Comment:
For reference: https://github.com/albburtsev/bem-cn#proptypes-warnings

I'm going to wait for other maintainers to respond. I personally feel like option #2 can solve this problem in user-land and this PR is unnecassary. Is there any reason you couldn't just call `toString()` on your side? instead of expecting enzyme to do it?